apps/DemoApp/DemoAppUITests/DemoAppUITests.m

```diff
@@ -67,6 +67,36 @@ - (void)testIMASDK {
     }
 }
 
+- (void)testAVQueuePlayer {
+    XCUIApplication *app = [[XCUIApplication alloc] init];
+    [app setLaunchEnvironment:@{@"ENV_KEY": @"tr4q3qahs0gflm8b1c75h49ln", @"TEST_SCENARIO": @"AV_QUEUE"}];
```

Comment:
your ENV key shouldn't be hardcoded in the tests

Reply:
This is not my environment key is the one all tests are using , it's this dashboard https://dashboard.mux.com/organizations/ffhei0/environments/k0vjj8/metrics but we or at least I didn't have access. @nidhik was going to look into this to see if we could get permissions. But yeah all tests are targeting that env.